### PR TITLE
Add confirmation page

### PR DESCRIPTION
**Pull request by Puneet-here**
### Fixed Issues

$ https://github.com/Expensify/App/issues/13151
PROPOSAL: https://github.com/Expensify/App/issues/13151#issuecomment-1359691865

### Tests

1. Go to concierge > Click call icon
2. Fill data > Request call
3. Verify you see a confirmation page with fireworks gif
---
1. Go to settings > security 
2. Change password > Fill data > save
3. Verify you see a confirmation page with fireworks gif
---
1. Go to settings > payments > Add payment method
2. Bank account > Fill data to add the bank account
3. Verify you see a confirmation page with fireworks gif
---
1. Import and render [TransferBalancePage](https://github.com/Expensify/App/blob/main/src/pages/settings/Payments/TransferBalancePage.js) in some other component (for example ProfilePage, don't forget to remove the code for profile page)
2. Return true in the if statement 
https://github.com/Expensify/App/blob/3ecec61a503dd180f6d302d6f565f6ee4f114825/src/pages/settings/Payments/TransferBalancePage.js#L155
3. Go to the page where you rendered the TransferBalancePage
4. Verify you see a new gif for dark theme

---
1.  Import and render [ActivateStep](https://github.com/Expensify/App/blob/main/src/pages/EnablePayments/ActivateStep.js) in some other component 
2. Go to the page where you rendered the ActivateStep
3. Verify you see a loading gif and there's no button
4. Make goldWallet true https://github.com/Expensify/App/blob/3ecec61a503dd180f6d302d6f565f6ee4f114825/src/pages/EnablePayments/ActivateStep.js#L39
5. Verify you see the confirmation fireworks gif and a button 
Note: At ios you will see that the header is near the status bar. It is because in ActivateStep the ScreenWrapper isn't present as the parent because the parent component of ActivateStep already contains a ScreenWrapper.
---
1. Go to settings > workspaces > 
2. Open a workspace > Connect bank account
3. Choose connect manually > Fill data > Save
8. Fill company info form and save
9. Verify you see a new gif for dark theme (Note: you will see the loading indicator first, the gif only shows up for brief moment)
- [x] Verify that no errors appear in the JS console

### Offline tests

No offline specific steps
### QA Steps

1. Go to concierge > Click call icon
5. Fill data > Request call
8. Verify you see a confirmation page with fireworks gif
---
1. Go to settings > security 
2. Change password > Fill data > save
3. Verify you see a confirmation page with fireworks gif
---
1. Go to settings > payments > Add payment method
2. Bank account > Fill data to add the bank account
3. Verify you see a confirmation page with fireworks gif
---
1. Go to settings > payments 
2. Transfer balance > go through the steps
3. Verify you see a confirmation page with fireworks gif
---
1. Go to settings > workspaces > 
2. Open a workspace > Connect bank account
3. Choose connect manually > Fill data > Save
9. Fill company info form and save
10. Verify you see a new gif for dark theme (Note: you will see the loading indicator first, the gif only shows up for brief moment)
- [x] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is correct English and approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

https://user-images.githubusercontent.com/101883770/209000410-0f10e9fd-3683-4e0b-a1de-3aa0189bee3a.mov

https://user-images.githubusercontent.com/101883770/209008565-45e59f86-9619-4d46-9408-4dc7abec051a.mov

Screenshot 2022-12-22 at 3 16 26 AM

Screenshot 2022-12-22 at 3 13 06 AM
Screenshot 2022-12-22 at 3 12 56 AM
Screenshot 2022-12-22 at 3 12 46 AM

</details>

<details>
<summary>Mobile Web - Chrome</summary>
Screenshot 2022-12-22 at 4 16 47 AM
Screenshot 2022-12-22 at 4 16 21 AM
Screenshot 2022-12-22 at 4 16 09 AM
Screenshot 2022-12-22 at 4 15 56 AM
Screenshot 2022-12-22 at 4 17 06 AM

</details>

<details>
<summary>Mobile Web - Safari</summary>
Screenshot 2022-12-22 at 3 30 51 AM
Screenshot 2022-12-22 at 3 30 34 AM

Screenshot 2022-12-22 at 3 30 24 AM
Screenshot 2022-12-22 at 3 30 03 AM
Screenshot 2022-12-22 at 3 27 46 AM

</details>

<details>
<summary>Desktop</summary>

Screenshot 2022-12-22 at 3 23 38 AM
Screenshot 2022-12-22 at 3 23 28 AM
Screenshot 2022-12-22 at 3 23 18 AM
Screenshot 2022-12-22 at 3 23 08 AM
Screenshot 2022-12-22 at 3 22 58 AM

</details>

<details>
<summary>iOS</summary>
Screenshot 2022-12-22 at 3 34 32 AM
Screenshot 2022-12-22 at 3 34 15 AM
Screenshot 2022-12-22 at 3 34 07 AM
Screenshot 2022-12-22 at 3 33 58 AM
Screenshot 2022-12-22 at 4 10 41 AM

</details>

<details>
<summary>Android</summary>
Screenshot 2022-12-22 at 4 18 38 AM
Screenshot 2022-12-22 at 4 18 27 AM
Screenshot 2022-12-22 at 4 18 16 AM
Screenshot 2022-12-22 at 4 18 05 AM
Screenshot 2022-12-22 at 4 17 57 AM

</details>

### CODE REVIEW

**Comment by Puneet-here** [comment]
I found a bug (not related to the PR) on pressing cross at confirmation screen of transfer balance page it navigates to the payments pages instead of closing the modal

**Comment by sobitneupane** [comment]
Thanks @Puneet-here. Changes are looking good. Will review it today.

**Review by luacmartins — DISMISSED** [review]
LGTM and tests well!

**Comment by mountiny** [comment]
@shawnborton Tagging you here for a review because I am jsut a bit concerneed about the background colour on some of the gifs. For example look at this: 
image

I am not sure if it was raised in other places, but I could not see it in the comments. The Hex of the gif green bg does not seem exactly the same as the app background. You can also check the screenshots in the PR body from Puneet and you can see the colour different there too.

Since the images are hosted in cloud we can update that without touching this PR. so I think it is NAB, but wanted to note this

**Comment by shawnborton** [comment]
Good catch. I think we might have to live with this until we replace these GIFs with Lottie animations, where we will have perfectly crisp SVG animations and no background color.

**Comment by shawnborton** [comment]
That being said, I'll flag this to Augenblick and see if they have any ideas!

**Comment by mountiny** [comment]
Great, thanks, wanted to get a confirmation this is not a blocker 🙇

**Review by sobitneupane — COMMENTED** [review]
### Screenshots/Videos
<details>
<summary>Web</summary>


https://user-images.githubusercontent.com/25876548/209203292-fa4cb0e0-8f10-4c70-94cd-9f436d486dca.mov

https://user-images.githubusercontent.com/25876548/209203342-052e2d43-e954-4439-80cd-4018fd5309ef.mov

https://user-images.githubusercontent.com/25876548/209203372-11f39c52-4bc2-460e-b0a7-df1b896fb090.mov

https://user-images.githubusercontent.com/25876548/209203388-f3c8f608-49a3-46f0-b327-eded14ff7a62.mov


https://user-images.githubusercontent.com/25876548/209203407-58a024a6-59fe-4d68-bfc2-da5c1dacbd84.mov


https://user-images.githubusercontent.com/25876548/209203381-40e0a00f-1418-44ef-8dee-30313f5e7c68.mov

https://user-images.githubusercontent.com/25876548/209203367-caff1dde-c7aa-42d4-ad39-cd62309abf6e.mov




</details>

<details>
<summary>Mobile Web - Chrome</summary>


https://user-images.githubusercontent.com/25876548/209203520-090abf42-24fa-4261-a53a-9a122aeb728a.mov


https://user-images.githubusercontent.com/25876548/209203529-446ea148-3eed-4017-8f14-d867484c8e26.mov


https://user-images.githubusercontent.com/25876548/209203533-f1961eb0-29fc-4cf9-9b97-fcf4503b18cf.mov


https://user-images.githubusercontent.com/25876548/209203536-75100239-8621-4f95-817f-e2dc1e7b371e.mov


https://user-images.githubusercontent.com/25876548/209204113-3d827174-6f2f-4f52-9e97-734259353e10.mov


</details>

<details>
<summary>Mobile Web - Safari</summary>

https://user-images.githubusercontent.com/25876548/209259620-0ddf1da7-c3f7-426c-a621-0aeae38f5009.mov


https://user-images.githubusercontent.com/25876548/209259639-307a14f8-89c3-4cb9-b361-2a3cb5a28310.mov




</details>

<details>
<summary>Desktop</summary>

https://user-images.githubusercontent.com/25876548/209203580-a5299524-127c-4cc2-94da-dec76e8620bb.mov


https://user-images.githubusercontent.com/25876548/209203639-24d56c2e-0050-47d4-839a-23e54d059570.mov


https://user-images.githubusercontent.com/25876548/209203644-39b2ab70-e10a-462f-9d4f-7d45019c3bcb.mov


https://user-images.githubusercontent.com/25876548/209203653-8127920e-c20e-4922-9119-2a596c551fcb.mov



</details>

<details>
<summary>iOS</summary>


https://user-images.githubusercontent.com/25876548/209203755-5e24ebb9-2866-4f8a-ae3a-4f4b7b3c1f55.mov

https://user-images.githubusercontent.com/25876548/209203817-ec78bff2-8c70-4051-9e27-4f1838f2ca8e.mov


https://user-images.githubusercontent.com/25876548/209203838-153d014e-ff08-4ba7-b338-5416b6e123cc.mov

https://user-images.githubusercontent.com/25876548/209203807-818be836-b3d3-41c7-9afa-4336b588ceba.mov


</details>

<details>
<summary>Android</summary>

https://user-images.githubusercontent.com/25876548/209203683-c3a17d59-ec17-45dc-9029-c45298419ebd.mov




https://user-images.githubusercontent.com/25876548/209203673-89144350-0a29-4e00-9a79-ae42b4429a17.mov

https://user-images.githubusercontent.com/25876548/209203692-16a72e89-81f2-49d5-a925-9af60ba69598.mov


https://user-images.githubusercontent.com/25876548/209203697-d47d440e-fc6c-4aca-94ae-7048ad254591.mov


</details>

**Comment by shawnborton** [comment]
Nice, these look great 🎆

**Review by mountiny — COMMENTED** [review]
@Puneet-here One minor thing

**Comment by shawnborton** [comment]
Okay we have two new gifs we can try to fix the BG issue that @mountiny spotted, @Puneet-here can you test with these locally and show us some screenshots? If these look better, I will upload to Cloudfront.

[animations.zip](https://github.com/Expensify/App/files/10290191/animations.zip)

**Comment by Puneet-here** [comment]
@shawnborton, it still doesn't match completely with the background

https://user-images.githubusercontent.com/101883770/209223826-f23ceb57-b775-4745-a2d9-1782974605d4.mov

**Comment by shawnborton** [comment]
Okay no worries, appreciate you giving that a shot! I say we just keep what we have for now and we'll revisit these with Lottie down the road.

File: src/components/ConfirmationPage.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        {props.shouldShowButton && (
+        <FixedFooter>
+            <Button
+                success
+                text={props.buttonText}
+                style={styles.mt6}
+                pressOnEnter
+                onPress={props.onButtonPress}
+            />
+        </FixedFooter>
```
[the remark this answers is not in the document]
Comment (mountiny) [reply src/components/ConfirmationPage.js:68]: @Puneet-here You around to update the indentation? we could merge it now if that would be updated (and linter wont hate it)

[the remark this answers is not in the document]
Comment (Puneet-here) [reply src/components/ConfirmationPage.js:68]: I am getting the same results and it is also same here 
https://github.com/Expensify/App/blob/9106261c29c16478acf3185907713198119dcc2c/src/pages/settings/Payments/AddPayPalMePage.js#L83-L85

[the remark this answers is not in the document]
Comment (mountiny) [reply src/components/ConfirmationPage.js:68]: I think it is mostly this way though:
https://github.com/Expensify/App/blob/9106261c29c16478acf3185907713198119dcc2c/src/pages/signin/SignInPageLayout/index.js#L41-L43

https://github.com/Expensify/App/blob/9106261c29c16478acf3185907713198119dcc2c/src/pages/signin/SignInPageLayout/SignInPageContent.js#L72-L78

https://github.com/Expensify/App/blob/9106261c29c16478acf3185907713198119dcc2c/src/pages/workspace/WorkspaceInitialPage.js#L212-L235

https://github.com/Expensify/App/blob/9106261c29c16478acf3185907713198119dcc2c/src/pages/workspace/WorkspaceMembersPage.js#L286-L294

https://github.com/Expensify/App/blob/9106261c29c16478acf3185907713198119dcc2c/src/pages/workspace/bills/WorkspaceBillsPage.js#L31-L36

[the remark this answers is not in the document]
Comment (Puneet-here) [reply src/components/ConfirmationPage.js:68]: Oh, you meant after the condition. Got it.

[the remark this answers is not in the document]
Comment (Puneet-here) [reply src/components/ConfirmationPage.js:68]: Found that we are using both ways but mostly there are spaces.

[the remark this answers is not in the document]
Comment (Puneet-here) [reply src/components/ConfirmationPage.js]: Done!

**Comment by mountiny** [comment]
Thank you for updating that, I see this is not super enforced 🙇 

We are missing a checklist from @sobitneupane :/

**Review by mountiny — APPROVED** [review]

**Review by luacmartins — APPROVED** [review]

**Review by sobitneupane — APPROVED** [review]
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified tests pass on **all platforms** & I tested again on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is correct English and approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

**Review by mountiny — APPROVED** [review]
Thank you guys @Puneet-here @sobitneupane 🙇

**Comment by mvtglobally** [comment]
@mountiny @shawnborton Can you pls validate the following steps internally? We do not have active wallet to be able to transfer. Rest steps we are QA-ing

1. Go to settings > payments
2. Transfer balance > go through the steps
3. Verify you see a confirmation page with fireworks gif

**Comment by AndrewGable** [comment]
I see fireworks on the transfer page!

Screenshot 2022-12-28 at 11 28 16 AM
Screenshot 2022-12-28 at 11 27 39 AM

**Comment by Santhosh-Sellavel** [comment]
Seems this PR caused a [regression](https://github.com/Expensify/App/issues/14734), i.e missed catching this bug where the welcome text is not centered in Spanish!

**Comment by sobitneupane** [comment]
Thanks @Santhosh-Sellavel for linking the issue. Though the bug precedes this PR and was existent in some confirmation screens like `PasswordConfirmationScreen` and `RequestCallConfirmationScreen`, It should have been caught in this PR.

**Pull request merged**